src/component/directive.ts

```diff
@@ -271,13 +271,39 @@ QWeb.addDirective({
     let tattClass = node.getAttribute("t-att-class");
     let styleAttr = node.getAttribute("style");
     let tattStyle = node.getAttribute("t-att-style");
+
+    const dataAttrs = [...node.attributes].filter(a => /^(t-attf?-)?data-.+$/.test(a.name)).sort();
```

Comment:
sort?

Reply:
This needs to be discussed.
What to do when template looks like this:
```xml
<Component t-att-data-foo="var" data-foo="42" t-attf-data-foo="var is {{var}}" />
```

Sorting is not the solution, though :-D